.github/workflows/build.yaml

```diff
@@ -4,9 +4,10 @@ on:
   workflow_dispatch:
   push:
     branches:
-      - 'main'
+      - "main"
```

Comment:
@flozi00 can you revert changes to `build.yaml`?

Reply:
done